training/common/usr/lib/systemd/system/upgrade-informer.timer

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Runs upgrade informer periodically
+ConditionPathExists=/run/ostree-booted
+
+[Timer]
+OnBootSec=1h
+# This time is relatively arbitrary and obviously expected to be overridden/changed
```

Comment:
dont want to encourage people to change this, so please drop the comment. usage of OnBootSec is ok.